app-i18n/koreanim/patches/koreanim-0.0.1.patchset

```diff
@@ -0,0 +1,25 @@
+From abcdefghijklmnopqrstuvwxyz0123456789 Sun Apr 9 00:00:00 2022
+From: Terra Yi <terrayi@gmail.com>
+Date: Sun, 9 Apr 2022 01:32:00 +1200
+Subject: Build fix for Haiku
+
+
+diff --git a/Makefile b/Makefile
+index 92eab78..20474ef 100644
+--- a/Makefile
++++ b/Makefile
+@@ -19,9 +19,9 @@ LINKER_FLAGS=
+ 
+ all: _APP_ default
+ 
+-include /boot/develop/etc/makefile-engine
++include /boot/system/develop/etc/makefile-engine
```

Comment:
There is a variable around that fixes detection:
https://github.com/HaikuArchives/ResourceEdit/blob/c1c0d1ff64fc8c86b06252c23deed84f07711056/source/Makefile#L35
PS you could upstream this to your repository?